evap/results/views.py

```diff
@@ -114,15 +112,10 @@ def course_detail(request, semester_id, course_id):
                 course_sections_top.append(section)
         else:
             contributor_sections.setdefault(section.contributor,
-                                            {'total_votes': 0, 'sections': []})['sections'].append(section)
-
-            for result in section.results:
-                if isinstance(result, TextResult):
-                    contributor_sections[section.contributor]['total_votes'] += 1
-                elif isinstance(result, RatingResult) or isinstance(result, YesNoResult):
-                    # Only count rating results if we show the grades.
-                    if course.can_publish_rating_results:
-                        contributor_sections[section.contributor]['total_votes'] += result.total_count
+                                            {'has_votes': False, 'sections': []})['sections'].append(section)
+
+            if any(result.question.is_rating_question and result.total_count or result.question.is_text_question for result in section.results):
```

Comment:
i'm not 100% sure i understand, but rating answers that cannot be published never make it out of `collect_results`, so `has_votes` should be `False` in your case. could you verify that?

Reply:
i see. now we have the following situation: in case only one user votes for a course, the rating results are not published. staff and contributors can still access the results page and for every question the number of people that voted for it is shown to be 0 even if the one person actually voted for it.
so this changed compared to what we had before where the shown number of voters was always the actual number of voters and only the distribution and average bar didn't show values when the rating results couldn't be published. can we get that back working like that?